packages/picasso/src/AccordionSummary/AccordionSummary.tsx

```diff
@@ -12,9 +12,9 @@ export interface Props extends StandardProps, ButtonOrAnchorProps {
 }
 
 // We can't create here intermediate object for AccordionSummary
-// because MUI ExpansionPanel use type check to set Summary in the
+// because MUI Accordion use type check to set Summary in the
 // correct place of the markdown
-// https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/ExpansionPanel/ExpansionPanel.js#L144
-export default withStyles(styles)(MUIAccordionSummary) as FunctionComponent<
-  Props
->
+// https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Accordion/Accordion.js#L144
```

Comment:
URL to accordion has changed, although not sure how useful it is

```suggestion
// https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Accordion/Accordion.js#L144
```

Reply:
Wow, nice 👍 I will check and update the link if needed.